specification/monitor/resource-manager/readme.md

```diff
@@ -31,6 +31,15 @@ openapi-type: arm
 tag: package-2019-06
 ```
 
+### Tag: package-2020-01-01-preview
+
+These settings apply only when `--tag=package-2020-01-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2020-01-01-preview'
+input-file:
+- Microsoft.Insights/preview/2020-01-01-preview/managementGroupDiagnosticSettings_API.json
+```
```

Comment:
@MyronFanQiu Please review if this is the right way to add a new api-version.

Would the better way be to add a new package with all the existing api-versions from the previous package, and only add the new api-version to that. so something like this:

``` yaml
title: MonitorClient
description: Monitor Management Client
openapi-type: arm
tag: package-2020-06
```

 

### Tag: package-2020-06

 

These settings apply only when `--tag=package-2020-06` is specified on the command line.

 

```yaml $(tag) == 'package-2020-06'
input-file:
- Microsoft.Insights/stable/2015-04-01/autoscale_API.json
- Microsoft.Insights/stable/2015-04-01/operations_API.json
- Microsoft.Insights/stable/2016-03-01/alertRulesIncidents_API.json
- Microsoft.Insights/stable/2016-03-01/alertRules_API.json
- Microsoft.Insights/stable/2016-03-01/logProfiles_API.json
- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettings_API.json
- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettingsCategories_API.json
- Microsoft.Insights/stable/2019-06-01/actionGroups_API.json
- Microsoft.Insights/stable/2017-04-01/activityLogAlerts_API.json
- Microsoft.Insights/stable/2015-04-01/activityLogs_API.json
- Microsoft.Insights/stable/2015-04-01/eventCategories_API.json
- Microsoft.Insights/stable/2015-04-01/tenantActivityLogs_API.json
- Microsoft.Insights/stable/2018-01-01/metricDefinitions_API.json
- Microsoft.Insights/stable/2018-01-01/metrics_API.json
- Microsoft.Insights/preview/2017-11-01-preview/baseline_API.json
- Microsoft.Insights/preview/2017-11-01-preview/calculateBaseline_API.json
- Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json
- Microsoft.Insights/stable/2018-03-01/metricAlert_API.json
- Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json
- Microsoft.Insights/preview/2017-12-01-preview/metricNamespaces_API.json
- Microsoft.Insights/preview/2018-11-27-preview/vmInsightsOnboarding_API.json
- Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json
- Microsoft.Insights/preview/2020-01-01-preview/managementGroupDiagnosticSettings_API.json
```


If this is the right way, what should be the change in the readme.python.md file?

Reply:
@changlong-liu  can you review this comment?

Reply:
Hi @admukhty , I think it's fine so far for Python in readme.md.  And the change in readme.python.md should be just in the same way what you do in this PR.